subprojects/common/kotlin-ast-parser/src/main/kotlin/com/avito/instrumentation/impact/KotlinClassesFinder.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.instrumentation.impact
+
+import java.io.File
+
+interface KotlinClassesFinder {
+
+    fun find(file: File): List<String>
+
+    fun find(projectDir: File, relativePath: FilePath): List<Regex>
```

Comment:
// todo try to reuse  ` fun find(file: File): List<String>` here

Reply:
i extracted regex transformation logic and moved to impact analysus call site; there is no need to know about regexes here